Makefile

```diff
@@ -5,9 +5,9 @@ all: get-deps code-analysis test
 config-install:
 	mkdir -p "$(HOME)/.config/perun"
 	cp defaults/main.yaml "$(HOME)/.config/perun/main.yaml"
-	mkdir -p "$(HOME)/perun"
-	cp defaults/blocked.json "$(HOME)/perun/blocked.json"
-	cp defaults/unblocked.json "$(HOME)/perun/unblocked.json"
+	mkdir -p "$(HOME)/.config/perun/stack-policies"
```

Comment:
Line 6 and 8 are kinda redundant - `-p` creates all subdirectories that does not exist on a given path.